src/app/technical-record/technical-record.component.html

```diff
@@ -6,27 +6,54 @@
     vtmFormConnector="vehicleRecordForm"
   >
     <h1 class="govuk-heading-xl title">Technical record</h1>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-third govuk-!-margin-right-0 govuk-!-padding-right-0">
+        <div *ngIf="!createState && !editState">
+          <button
+            id="test-change-btn"
+            class="govuk-button"
+            type="button"
+            (click)="editTechRecord()"
+          >
+            Change technical record
+          </button>
+        </div>
 
-    <div *ngIf="!createState && !editState">
-      <button id="test-change-btn" class="govuk-button" type="button" (click)="editTechRecord()">
-        Change technical record
-      </button>
-    </div>
+        <ng-container>
+          <button
+            id="test-save-btn"
+            class="govuk-button"
+            type="submit"
+            *ngIf="editState || createState"
+          >
+            Save technical record
+          </button>
+          <a id="cancel2" (click)="cancelTechRecord()" *ngIf="editState">
+            <span>Cancel</span>
+          </a>
+        </ng-container>
+      </div>
 
-    <ng-container>
-      <button
-        id="test-save-btn"
-        class="govuk-button"
-        type="submit"
-        *ngIf="editState || createState"
+      <ng-container
+        *ngIf="
```

Comment:
lets pick up the discussion on this. Also we need to ask for the visibility of this button in edit & create mode? Same goes for the plate button when you rebase this branch. Should the plate button be visible in edit mode